cyflash/bootload.py

```diff
@@ -206,6 +277,17 @@ def progress(self, message=None, current=None, total=None):
 
 def main():
 	args = parser.parse_args()
+
+	if (args.logging_config):
+		import sys
```

Comment:
What's "import sys" for, here? I don't believe "import logging" is necessary, either.